gcs-avro-tools.rb

```diff
@@ -0,0 +1,22 @@
+class GcsAvroTools < Formula
+  desc "GCS compatible Apache Avro Tools"
+  homepage "https://github.com/spotify/gcs-tools"
+  url "https://github.com/spotify/gcs-tools/releases/download/v0.1.1/avro-tools-1.8.1.jar"
+  sha256 "382f2bfd756b190e4bad5038807705ae66dda132e17a0ac88a0069f246e720d0"
+  version "0.1.1"
```

Comment:
@nevillelyh fyi - version formulae by `gcs-tools` version NOT the `avro`/`parquet` tool version. what do you think?